playbooks/common/openshift-cluster/upgrades/post.yml

```diff
@@ -56,3 +56,16 @@
       {{ oc_cmd }} patch dc/docker-registry -n default -p
       '{"spec":{"template":{"spec":{"containers":[{"name":"registry","image":"{{ registry_image }}"}]}}}}'
       --api-version=v1
+
+# Check for warnings to be printed at the end of the upgrade:
+- name: Check for warnings
+  hosts: oo_masters_to_config
+  tasks:
+  # Check if any masters are using pluginOrderOverride and warn if so, only for 1.3/3.3 and beyond:
+  - command: >
+      grep pluginOrderOverride {{ openshift.common.config_base }}/master/master-config.yaml
+    register: grep_plugin_order_override
+    when: openshift.common.version_gte_3_3_or_1_3 | bool
+  - name: Warn if pluginOrderOverride is in use in master-config.yaml
+    debug: msg="WARNING pluginOrderOverride is being deprecated in master-config.yaml, please see https://docs.openshift.com/enterprise/latest/architecture/additional_concepts/admission_controllers.html for more information."
```

Comment:
We really should avoid the use of `pause` unless we are working around a timing issue. Relying on a user to see the output is too risky for things that need action.

Ansible does support sending back warnings from modules, but that is part of a particular module, not general feedback to the user.

What we really need is a filter plugin that tracks these things along the way and then outputs additional info to the user after the run has completed.